### ISSUE TITLE
Downloading is broken

### ISSUE THREAD

**Issue by thesby** [yajiedesign/mxnet#2]
Downloading of pre-build mxnet is often broken.  Is it possible to download from other source, such as `baidu cloud` ?

**Comment by chrinide** [comment]
Yes, It is very difficult to download these files. Any other cloud alternative?

**Comment by yajiedesign** [comment]
this because gfw....
and baidu cloud not way auto upload.

**Comment by thesby** [comment]
Yes, but a vip can upload auto.
